provisions/roles/repo_tracking/tasks/main.yml

```diff
@@ -69,6 +69,7 @@
   sudo: yes
   tags: repo_tracking
 
+
```

Comment:
unrelated changes

Reply:
I was about to add the manage.py here but decided against. Should have removed it